package Exception;

public class FinallyClause  {

   public static void main(String[] args){
	   int a=3;
	   int b=4;
	   try{
	   System.out.println(a/b);
	   }
	   
	   catch (ArithmeticException e){
		   System.out.println("problem in devision variable");
	   }
	   finally{
	    	  System.out.println("In Finqally");
	    	  
	   }
   }
}
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  
	    	  